r2-core/src/main/java/com/linkedin/r2/transport/http/common/HttpBridge.java

```diff
@@ -209,6 +223,12 @@ public void onResponse(TransportResponse<StreamResponse> response)
     };
   }
 
+  private static boolean shouldReturnRetriableRequestException(Throwable responseError)
+  {
+    return responseError instanceof StreamException && responseError.getMessage()
```

Comment:
Is this the right StreamException? Shouldn't it be [Http2Exception.StreamException](https://github.com/netty/netty/blob/70d6a3f40d7e8fd3f5ced7600ed209c58944f673/codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Exception.java#L267)?